.github/workflows/connector-ui-sanity-tests.yml

```diff
@@ -129,23 +130,46 @@ jobs:
           DATABASE_URL: postgres://db_user:db_pass@localhost:5432/hyperswitch_db
         run: diesel migration run
 
-      - name: Start server and run tests
+      - name: Build project
         if: ${{ (github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name) }}
-        env:
-          UI_TESTCASES_PATH: ${{ secrets.UI_TESTCASES_PATH }}
-          INPUT: ${{ matrix.connector }}
-        shell: bash
-        run: sh .github/scripts/run_ui_tests.sh
+        run: cargo build --package router --bin router
 
-      - name: View test results
+      - name: Start server
         if: ${{ (github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name) }}
+        env:
+          BROWSER_DATA_PASSPHRASE: ${{ secrets.CONNECTOR_AUTH_PASSPHRASE }}
+          INPUT: ${{ matrix.connector }}
+          UI_TESTCASES_PATH: ${{ secrets.UI_TESTCASES_PATH }}
         shell: bash
-        run: cat tests/test_results.log
+        run: .github/scripts/run_ui_tests.sh
```

Comment:
If the script only starts the server for running UI tests but doesn't actually run the tests, we'll have to name the script suitably.

Reply:
I actually wanted to get rid of this instead of just renaming the file. What do you say?